packages/remark-iframes/src/index.js

```diff
@@ -96,49 +42,51 @@ module.exports = function plugin (opts) {
       })
     }
 
-    let finalUrl, thumbnail, fallback
+    let finalUrl, thumbnail
+    const data = {
+      hName: provider.tag || 'iframe',
+      hProperties: {
+        src: 'tmp',
+        width: provider.width,
+        height: provider.height,
+        allowfullscreen: true,
+        frameborder: '0',
+      },
+    }
 
     if (provider.oembed) {
-      const reqUrl = `${provider.oembed}?format=json&url=${encodeURIComponent(url)}`
-
-      await makeHttpRequest(reqUrl).then((oembedRes) => {
-        finalUrl = oembedRes.url
-        thumbnail = oembedRes.thumbnail
-
-        if (!provider.height) provider.height = oembedRes.height
-        if (!provider.width) provider.width = oembedRes.width
-        if (!provider.tag) provider.tag = 'iframe'
-      }, (e) => {
-        fallback = true
+      Object.assign(data, {
```

Comment:
We add useful metadata to the iframe AST node when we encounter an oembed, we'll use it later on